examples/oft-solana/tasks/solana/retryPayload.ts

```diff
@@ -24,7 +24,7 @@ interface Args {
     withPriorityFee: number
 }
 
-task('lz:oft:solana:clear', 'Clear a stored payload on Solana')
+task('lz:oft:solana:retryPayload', 'Retry a stored payload on Solana')
```

Comment:
NIT: can we use kebab case instead? 

`retry-payload`

to be consistent with other tasks

e.g.

`lz:oft:solana:get-rate-limits`

`lz:oft:solana:outbound-rate-limit`

Reply:
done in the rebased PR